client/src/store/index.js

```diff
@@ -1233,6 +1233,20 @@ mutations: {
         .catch(error => console.log('ERROR: ' + error))
     },
 
+    syncNR({dispatch,commit},nrNumber) {
+      console.log('action: syncing data for nr number: ' + nrNumber + ' from with nro')
+      const myToken = localStorage.getItem('KEYCLOAK_TOKEN')
+      const url = '/api/v1/requests/' + nrNumber + '/editclosed'
```

Comment:
Per discussion, call the API syncNR instead of editclosed, since that's misleading.